
// Erros podem ser gerados usando
// a instrução throw, enquanto o bloco try é usado para agrupar código que pode gerar erros, e o bloco catch trata o erro, evitando que o programa trave.

// try {
// alert("Bem vindo!");
//} catch (err) {
//  console.log('Erro:', err);
//}

// try {
// alert("Bem vindo!");
//} catch (err) {
//  throw new Error('Erro', err);
//} finally {
//  
//}